scripts/README.md

```diff
@@ -0,0 +1,123 @@
+# Training Dataset Export Scripts
+
+This directory contains scripts for converting human-graded code generation evaluations into industry-standard training datasets for LLM fine-tuning and research.
```

Comment:
Now this is quite a surprising and interesting idea. Wow. This could open the door to fine tuning models - amazing! I wonder if this could also unlock A/B testing for the prompt templates.